cli/lsp/task_pool.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+use crossbeam_channel::Sender;
+
+pub struct TaskPool<T> {
+  sender: Sender<T>,
+}
```

Comment:
Now that this isn't using threadpool, this struct seems not so useful. I suggest placing `sender: Sender<Task>` directly into `ServerState`. I think the `Handle` abstraction can also be removed.

Reply:
I removed the module and the `Handle` abstraction.